content/en/containers/cluster_agent/admission_controller.md

```diff
@@ -128,13 +128,47 @@ Finally, run the following commands:
 - `kubectl apply -f agent-services.yaml`
 - `kubectl apply -f cluster-agent-deployment.yaml`
 
-### APM and DogStatsD
 
-To configure DogstatsD clients and APM tracers automatically, inject the environment variables `DD_AGENT_HOST` and `DD_ENTITY_ID` by using one of the following:
+### APM
+You can configure the Cluster Agent (version 7.39 and higher) to inject APM tracing libraries automatically.
 
+After you install the Cluster Agent, do one of the following:
 - Add the label `admission.datadoghq.com/enabled: "true"` to your pod.
 - Configure the Cluster Agent admission controller by setting `mutateUnlabelled` (or `DD_ADMISSION_CONTROLLER_MUTATE_UNLABELLED`, depending on your configuration method) to `true`.
 
+To opt-in your container for library injection, use Pod annotations inside your application's YAML file to specify language tracers and versions.
+
+
+The annotations are a `key: value` pair in the following format:
+
+```yaml
+datadoghq.com/<language>-lib.version: <lib-version>
+```
+
+Adding a this annotation results in the injection of the tracer library for that language and version into the containerized application.
+Valid `<language>` values are:
+- `java`
+- `js`
+- `python`
+
+For example to inject the latest Java tracer:
+
+```yaml
+annotations:
+    datadoghq.com/java-lib.version: "latest"
```

Comment:
You need to configure the label `admission.datadoghq.com/enabled: "true"` in case `mutateUnlabelled` is set to `false` .